specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/examples/PrivateEndpointConnections_Create_MaximumSet_Gen.json

```diff
@@ -2,20 +2,20 @@
   "title": "PrivateEndpointConnections_Create",
   "operationId": "PrivateEndpointConnections_Create",
   "parameters": {
-    "subscriptionId": "5ED36C6E-F804-407F-AD4D-126277AB3EEB",
+    "subscriptionId": "A809C6A2-DF28-4BEE-8440-E25396838122",
```

Comment:
I would suggest using `{subscriptionId}` instead of the real subscription ID in the example, to hide credential.